standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestObjectStore.java

```diff
@@ -1124,6 +1125,40 @@ public void testEmptyTrustStoreProps() {
     setAndCheckSSLProperties(true, "", "", "jks");
   }
 
+  /**
+   * Tests getPrimaryKeys() when db_name isn't specified.
+   */
+  @Test
+  public void testGetPrimaryKeys() throws Exception {
```

Comment:
Please add this Test to class TestPrimaryKey.java

Reply:
The change itself is in ObjectStore, and the API too is in ObjectStore. The `TestPrimaryKey` is a parametrised test and for one case, this issue doesn't happen. I think the test is better here.

Reply:
You are right that its part of ObjectStore but TestPrimaryKey class was created so that we can have all sort of test for primary key in one place.